tests/test_parametric_shapes/test_SweepStraightShape.py

```diff
@@ -0,0 +1,87 @@
+
+import unittest
+
+import pytest
+from paramak import RotateStraightShape, SweepStraightShape
+
+
+class TestSweepStraightShape(unittest.TestCase):
+
+    def setUp(self):
+        self.test_shape = SweepStraightShape(
+            points=[(-10, 10), (10, 10), (10, -10), (-10, -10)],
+            path_points=[(50, 0), (30, 50), (70, 100), (50, 150)]
+        )
+
+    def test_default_parameters(self):
+        """Checks that the default parameters of a SweepStraightShape are correct."""
+
+        # assert self.test_shape.rotation_angle == 360
+        assert self.test_shape.stp_filename == "SweepStraightShape.stp"
+        assert self.test_shape.stl_filename == "SweepStraightShape.stl"
+        assert self.test_shape.azimuth_placement_angle == 0
+        assert self.test_shape.workplane == "XY"
+        assert self.test_shape.path_workplane == "XZ"
+        assert self.test_shape.force_cross_section == False
+
+    def test_absolute_shape_volume(self):
+        """Creates a SweepStraightShape and checks that the volume is correct."""
+
+        self.test_shape.path_points = [(50, 0), (50, 50), (50, 100)]
+        assert self.test_shape.solid is not None
+        assert self.test_shape.volume == pytest.approx(20 * 20 * 100)
+
+    def test_relative_shape_volume(self):
+        """Creates two SweepStraightShapes and checks that their relative volumes
+        are correct."""
+
+        test_volume = self.test_shape.volume
+        self.test_shape.points = [(-20, 20), (20, 20), (20, -20), (-20, -20)]
+        assert self.test_shape.volume == pytest.approx(test_volume * 4)
+
+    def test_relative_shape_volume(self):
```

Comment:
**BUG RISK**

This test has the same name as the last test, so the last test will be overridden.

Reply:
Fixed by https://github.com/ukaea/paramak/commit/3bcbcbf00500089c2d27ff60e1ceed14241cea8a